corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -773,7 +772,7 @@ def _record_timing(self, status):
             'status_code': status,
             'device_type': 'webapps' if is_webapps else 'other',
         }
-        maybe_add_domain_tag(self.domain, tags)
+        tags['domain'] = self.domain
```

Comment:
nit: now that this is simpler logic, I think it makes more sense to put this into the initialization statement for tags above (alongside status_code and device_type)

Reply:
duh. fixed